subprojects/degrees-and-programs/source/static/README.md

```diff
@@ -0,0 +1,57 @@
+# chapman
```

Comment:
This README seems incomplete. Does this directory include tools to build source?